src/keywords.rs

```diff
@@ -98,6 +98,7 @@ define_keywords!(
     BEGIN,
     BEGIN_FRAME,
     BEGIN_PARTITION,
+    BERNOULLI,
```

Comment:
Bernoulli isn't reserved ([1](https://jakewheat.github.io/sql-overview/sql-2016-foundation-grammar.html#non-reserved-word)). Can't you match against it without making it a keyword?

[1] : https://jakewheat.github.io/sql-overview/sql-2016-foundation-grammar.html#non-reserved-word